config/Config-kernel.in

```diff
@@ -1166,6 +1172,13 @@ if KERNEL_IPV6
 	config KERNEL_LWTUNNEL_BPF
 		def_bool n
 
+        config KERNEL_MPTCP_IPV6
+                bool "Enable IPv6 MultiPath TCP support"
+                default y
```

Comment:
Is there even a valid use case where you have/want IPv6 and MPTCP support enabled, but don't want MPTCP for IPv6? If not, I'd suggest this being a `def_bool y` instead of a user option.